BDArmory/Control/IBDAIControl.cs

```diff
@@ -58,12 +58,14 @@ public interface IBDAIControl
 
         void CommandTakeOff();
 
+        void CommandFollowWaypoints();
+
         Vector3d commandGPS { get; }
         PilotCommands currentCommand { get; }
         ModuleWingCommander commandLeader { get; }
 
         #endregion WingCommander
     }
 
-    public enum PilotCommands { Free, Attack, Follow, FlyTo }
+    public enum PilotCommands { Free, Attack, Follow, FlyTo, Waypoints }
```

Comment:
This may not be strictly necessary. It was added to support a command mode, but we don't actually use the command mode.